packages/shared/src/components/cards/SharedPostCardFooter.tsx

```diff
@@ -2,39 +2,51 @@ import classNames from 'classnames';
 import React, { ReactElement } from 'react';
 import { Post } from '../../graphql/posts';
 import { cloudinary } from '../../lib/image';
+import { IconSize } from '../Icon';
+import { CardImage, CardVideoImage } from './Card';
 
 type SharedPostCardFooterProps = {
   isShort: boolean;
+  isVideoType?: boolean;
 } & Pick<Post, 'sharedPost'>;
 
 export const SharedPostCardFooter = ({
   sharedPost,
   isShort,
+  isVideoType,
 }: SharedPostCardFooterProps): ReactElement => {
+  const ImageComponent = isVideoType ? CardVideoImage : CardImage;
   return (
     <div
       className={classNames(
-        'flex flex-1 gap-2 p-3 mb-2 rounded-12 border border-theme-divider-tertiary',
+        'p-3 gap-3 mb-2 rounded-12 border border-theme-divider-tertiary flex w-fit max-h-[13.5rem] flex-auto h-auto min-h-[0]',
         isShort ? 'flex-row items-center' : 'flex-col',
       )}
     >
-      <span
+      <div
         className={classNames(
-          'typo-footnote text-theme-label-secondary',
-          isShort ? 'line-clamp-4 flex-1' : 'line-clamp-2',
+          'typo-footnote text-theme-label-secondary flex',
+          isShort ? 'line-clamp-4 pr-3 w-8/12' : 'line-clamp-2',
         )}
       >
         {sharedPost.title}
-      </span>
-      <div
-        className={classNames(
-          'rounded-xl',
-          isShort ? 'h-full aspect-square' : 'flex-1',
-        )}
-        style={{
-          background: `url(${sharedPost.image}) center center / cover, url(${cloudinary.post.imageCoverPlaceholder}) center center / cover`,
-        }}
-      />
+      </div>
+
+      <div className={classNames('flex flex-auto h-auto overflow-auto')}>
+        <ImageComponent
+          size={isShort ? IconSize.XLarge : IconSize.XXXLarge}
+          alt="Shared Post Cover image"
+          src={sharedPost.image}
+          fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+          className={classNames(
+            'object-cover h-auto min-h-[0px]',
```

Comment:
The should be a default better alternative for the min-h, also is it really neeeded?

Reply:
With my implementation, really needed. Allows the image to shrink in height when there is less room in the container. Can't really think of another way to do this at the moment, but happy to hear any other ideas.

If we don't like this and don't have any ideas, I'll have to go at it again and try to re write